internal/storage/firestore/firestore.go

```diff
@@ -0,0 +1,124 @@
+package firestore
+
+import (
+	"context"
+
+	"cloud.google.com/go/firestore"
+	firebase "firebase.google.com/go"
+	"github.com/kyleshepherd/discord-tk-bot/internal/storage"
+	"google.golang.org/api/iterator"
+	"google.golang.org/api/option"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type (
+	KillStore struct {
+		client *firestore.Client
+	}
+)
+
+func NewKillStore(ctx context.Context, projectID string) (*KillStore, error) {
+	conf := &firebase.Config{ProjectID: projectID}
+	opt := option.WithCredentialsFile("./service-account-file.json")
```

Comment:
One more thing, I'd imagine you'd want to pass this in as variable so it could be added to config in the future

Reply:
ah yeah good point!